README.md

```diff
@@ -1 +1,4 @@
-# opencraft-simple-theme
\ No newline at end of file
+# Opencraft Simple theme
```

Comment:
@giovannicimolin. I think it is better to drop `opencraft` from the repository name or name it something like `edx-simple-theme`.

Reply:
@lgp171188 I can do that. I've chosen this to leave some Opencraft branding on the name :stuck_out_tongue: